tests/main.cpp

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
I get the following error from all of the `tests/*.cpp` files:
> 1>C:\Users\jbspj\Documents\GitHub\NVIDIA\abstract-fps\tests\main.cpp(57,1): fatal error C1010: unexpected end of file while looking for precompiled header. Did you forget to add '#include "pch.h"' to your source?

Reply:
That's a problem. I thought I disabled precompiled headers. Maybe I forgot to change the dropdown box to release _and_ debug rather than just debug. Will update.